### ISSUE TITLE
Nova?

### ISSUE THREAD

**Issue by eugenefvdm** [ynacorp/nova-swatches#7]
I really like this package and I feel it's more powerful than the Nova 4 built-in one. I wish we could update to Nova 4.

**Comment by pekka** [comment]
Agreed! This package is beautiful! Best color picker I've seen for Nova.

Unfortunately, it seems to break in Nova 4, and I'm too new to the ecosystem to do a quick fix.
